examples/fabric/v4/cloudRouterConnectivity/cloudRouter2oracle/terraform.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_providers {
+    equinix = {
+      source = "equinix/equinix"
+    }
+  }
+}
```

Comment:
@srushti-patl Can we please add a new line to the end of file? A few more files below as well.

Reply:
Done